### PR TITLE
Add tool to modify installation image bootloader

### PR DESCRIPTION
**Pull request by jkonecny12**
This script is able to extract ``isolinux.cfg``, put it back and also injecting initrd overlay to the ISO image.

I hope this will be a helpful tool in debugging of ISO boot. At least it will be helpful for me.

### CODE REVIEW

File: modify_install_iso/lib/__init__.py [hunk 1]
```diff
@@ -50,3 +50,20 @@ def mount_iso(image):
...
+    :param str source_iso: source for modification
+    :param str output_iso: output modified iso
+    :param str graft_dir: path to the directory with content we want to modify
+    :param bool verbose: should the call to pungi be verbose to people
+    """
+    cmd = ["pungi-patch-iso"]
+
+    cmd.append(output_iso)
+    cmd.append(source_iso)
+    cmd.append(graft_dir)
```
Comment (VladimirSlavik) [inline modify_install_iso/lib/__init__.py]: It does not really matter, but I'm curious - why not `cmd = ["pungi-patch-iso", output_iso, source_iso, graft_dir]` ...?

File: modify_install_iso/create_dvd.py [hunk 2]
```diff
@@ -27,13 +27,7 @@
 
 
 def create_custom_dvd(source_iso, graft_dir, output_iso):
```
Comment (VladimirSlavik) [inline modify_install_iso/create_dvd.py]: This function just calls another function with the same parameters, so why keep it?

**Review by VladimirSlavik — APPROVED** [review]
Thank you, this will be very useful!

File: modify_install_iso/lib/__init__.py (hunk 1 above)
Comment (jkonecny12) [reply modify_install_iso/lib/__init__.py]: Don' really have a reason. I guess because I did it this way before on other places :D.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Renamed ``call_pungi_patch_iso`` to ``create_custom_dvd`` which is what is the goal here.
* Fix bug with incorrect permissions when writing the file.

**Pull request merged**